test/fixtures.json

```diff
@@ -518,7 +518,7 @@
             {
               "tagName": "fallback_address",
               "data": {
-                "address": "tb1dqqqsyqcyq5rqwzqfpg9scrgwpuqsyqcyy5mynw"
+                "address": "tb1zx93njcnp8qcnqde4vgckydpkv33rsdnpxccrwetp8qmn2vnrv4jnjdfs8q6rsdpksh0hkj"
```

Comment:
Why was this changed?

Reply:
The bitcoinjs-lib address validation is more strict in v6. It requires:
- version === 0 for bech32
- version !== 0 for bech32m
This fix uses an address that has version === 2 with bech32m encoding

Reply:
With the old `address` value plus `v6` an exception was thrown earlier (`'Fallback address type is unknown'`), and the code did get the chance to check and fail with `Fallback address witness version is unknown`